### PR TITLE
 Better user experience in connection with subscribing to (public) lists

### PR DESCRIPTION
**Pull request by b14sha**
https://platform.dandigbib.org/issues/3358

### CODE REVIEW

**Comment by cableman** [comment]
Please fix the Scrutinizer errors

File: modules/p2/ding_list/plugins/content_types/list_menu.inc [hunk 1]
```diff
@@ -59,19 +60,30 @@ function ding_list_list_menu_content_type_render($subtype, $conf, $panel_args, $
     }
 
     if (ding_list_allowed($list, DING_LIST_OPERATION_SHARE)) {
+      // Check if user owns the list.
       if (ding_list_user_has_access($list, 'owner', $user)) {
         $title = t('Share list');
       }
       else {
-        $title = t('Unfollow list');
+        // Tjeck if user allready follows list.
```
Comment (cableman) [inline modules/p2/ding_list/plugins/content_types/list_menu.inc]: There seams to be a typo here!

**Comment by JrgenGNielsen** [comment]
Please fix the Scrutinizer error

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**